### PR TITLE
Lims 4797 test case

### PR DESCRIPTION
**Pull request by modeso-NahlaMakhlouf**

### CODE REVIEW

File: github/workflows/CI.yml [hunk 1]
```diff
@@ -10,7 +10,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+<<<<<<< HEAD
+        test_cases: ["ui_testing/testcases/basic_tests/test06_orders.py:OrdersTestCases.test066_close_testplan_popup"]
+=======
         test_cases: ["ui_testing/testcases/basic_tests/test04_testunits.py:TestUnitsTestCases"]
+>>>>>>> 4e5068b766e626514d6a492041541f33ad1c3e80
```
[the remark this answers is not in the document]
Comment (modeso-NahlaMakhlouf) [reply github/workflows/CI.yml:17]: done

File: ui_testing/testcases/basic_tests/test06_orders.py [hunk 2]
```diff
@@ -2296,3 +2297,22 @@ def test065_download_suborder_sheet_for_single_order(self):
                                   f"{str(fixed_sheet_row_data)} : {str(formatted_orders[index])}")
             for item in formatted_orders[index]:
                 self.assertIn(item, fixed_sheet_row_data)
+
+
```
[the remark this answers is not in the document]
Comment (modeso-NahlaMakhlouf) [reply ui_testing/testcases/basic_tests/test06_orders.py:2301]: done

File: ui_testing/testcases/basic_tests/test06_orders.py [hunk 3]
```diff
@@ -2296,3 +2297,22 @@ def test065_download_suborder_sheet_for_single_order(self):
...
+
+    @parameterized.expand(['cancel_btn', 'close_btn'])
+    def test066_close_testplan_popup(self,button):
+        """ LIMS-4797- Make sure the user can press on the cancel button to close the pop-up or from the ( x ) sign
```
[the remark this answers is not in the document]
Comment (modeso-NahlaMakhlouf) [reply ui_testing/testcases/basic_tests/test06_orders.py:2305]: done

File: ui_testing/testcases/basic_tests/test06_orders.py [hunk 4]
```diff
@@ -2296,3 +2297,22 @@ def test065_download_suborder_sheet_for_single_order(self):
...
+                          """
+        order, payload = self.orders_api.create_new_order(materialTypeId=1)
```
[the remark this answers is not in the document]
Comment (modeso-NahlaMakhlouf) [reply ui_testing/testcases/basic_tests/test06_orders.py:2307]: done

File: ui_testing/testcases/basic_tests/test06_orders.py [hunk 5]
```diff
@@ -2296,3 +2297,22 @@ def test065_download_suborder_sheet_for_single_order(self):
...
+        self.orders_page.get_order_edit_page_by_id(id=order['order']['mainOrderId'])
+        self.base_selenium.click(element='order:testplan_popup_btn')
+        if button == 'cancel_btn':
+            self.base_selenium.wait_until_element_clickable(element='order:testplan_cancel_btn')
+            self.base_selenium.click(element='order:testplan_cancel_btn')
+        else:
+            self.base_selenium.wait_until_element_clickable(element='order:testplan_close_btn')
+            self.base_selenium.click(element='order:testplan_close_btn')
+            self.base_selenium.check_element_is_not_exist(element='order:testplan_popup')
+
```
[the remark this answers is not in the document]
Comment (modeso-NahlaMakhlouf) [reply ui_testing/testcases/basic_tests/test06_orders.py:2318]: done

File: github/workflows/CI.yml (hunk 1 above)
Comment (ghada-37) [inline github/workflows/CI.yml:17]: you need to remove this lines + line 13

**Review by ghada-37 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**